2017/_data/speakers.json

```diff
@@ -794,6 +794,33 @@
+      "biography": "Albert Margarit is a Software Development Manager and Architect at NetSuite. He dedicates most of his time to software design of robust and performant cloud solutions. He champions best coding practices and instills the culture of clean and reusable designs. Recently, he has been focused in designing a multi-tenant solution for backend processing. In his spare time he enjoys geeking out and playing inline hockey.",
+      "image": "assets/img/speakers/albert-margarit.png",
+      "ref": "albert-margarit",
+      "url": "infoSpeaker.html?ref=albert-margarit",
+      "twitter": "",
+      "homepage": "",
+      "talks": [
+        {
+          "type": "talk",
+          "title": "Keep calm and watch Kibana: proactive approach to software development",
+          "abstract": "<p class='abstract'>Imagine yourself building a highly-available and performant multi-tenant cloud system used by thousands of users. Your responsibility as an engineer or product manager in your company is to deliver value to your customers by cohesive and performant solutions.</p><p class='abstract'>Have you ever asked yourself any of the following questions?</br>How do I know that my team is delivering what our customers want? How much intuition do I use to develop requirements and solutions?</br>How do I ensure that our solutions are performant and scalable in all situations?</br>Am I sure that users are using our features with the frequency that we expected?</p><p class='abstract'>The common approach in software development is to adopt a reactive attitude towards the previous points: deliver the functionality to production, wait for customer feedback (if any) and just wait for issues and bugs to be raised by the customer. No bugs? No customer feedback? Everything is ok!</p><p class='abstract'>Unfortunately, the reactive approach is not scalable in an enterprise environment and will not help your team much to deliver valuable solutions to your customers.In the new cloud computing era, we have a great opportunity to change our mindset and move to a proactive approach where software development teams can immediately know what's going on in production and design new functionality based on customer's behavior and data. No more intuition-driven designs!</p><p class='abstract'>In this session, we will explore how feature instrumentation through ElasticSearch and Kibana can help cloud enterprises to change to a proactive approach: anticipate problems, get real insights about feature usage and empower their teams to own their features even after they are released to production. We will also explore how ElasticSearch can be used by multiple teams with the least friction possible and how to incorporate instrumentation as another milestone in the software development lifecycle just like requirements gathering or testing.Just think about it, keep calm and watch Kibana ;)</p>",
+          "tags": [
```

Comment:
I don't understand why you are adding this HTML Tag.
You can safely remove the HTML tag and just keep the text.